lib/provider/global_provider.dart

```diff
@@ -429,7 +405,8 @@ class GlobalProvider with ChangeNotifier {
   }
 
   List<LanguageData?> _languageDataList = [];
-  Map<String, String> _languageCodeMapper = {"eng": "English"};
+  Map<String, String> _codeToLanguageMapper = {"eng": "English"};
```

Comment:
this mapping we should get from a configuration file

Reply:
Map getting populated with language data we are getting from database.